### ISSUE TITLE
Movement of Dates on Nightly

### ISSUE THREAD

**Issue by Nosferican** [JuliaIO/JSON.jl#225]
Since Julia [0c0d6cc](https://github.com/JuliaLang/julia/commit/0c0d6cca5d3793eaf6321ef0577afd6ce594a9b5) JSON fails to compile due to `Writer.jl`
```
const IsPrintedAsString = Union{
    Dates.TimeType, Char, Type, AbstractString, Enum, Symbol}
```
which gives
```
ERROR: LoadError: LoadError: type Void has no field TimeType
```
Fix is to add the line to `Writer.jl`
```
import Dates
```

**Comment by KristofferC** [comment]
```
if VERSION >= v"0.7.0-DEV.2575"
    import Dates
end
```

**Comment by Nosferican** [comment]
Thanks!

**Comment by TotalVerb** [comment]
Closed by #226.
